BenchBnb/BenchBnb/Views/LogIn/Index.cshtml

```diff
@@ -1,6 +1,7 @@
-﻿@model BenchBnb.FormModels.CreateBench
+﻿
 @{
     ViewBag.Title = "Index";
 }
 
 <h2>Index</h2>
```

Comment:
I'd remove this view and the associated action method in the `LogInController` class.

Reply:
Fixed